lib/screens/map_view.dart

```diff
@@ -90,54 +88,31 @@ class _MapViewScreenState extends State<MapViewScreen> {
               Marker(
                   width: 110.0,
                   height: 110.0,
-                  point: latlng.LatLng(locationData?.latitude ?? 51.5,
-                      locationData?.longitude ?? -0.09),
+                  point: latlng.LatLng(locationData!.latitude!,
+                      locationData!.longitude!),
                   builder: (ctx) => const Avatar()),
-              spawnBeastie(),
-              spawnBeastie(),
-              spawnBeastie()
+              beastie1.spawnMarker(),
```

Comment:
So this will basically always spawn 3 beasties nearby wherever the user is? Not sure if I'm reading this right. If so, I'm wondering if it makes sense to add one more piece of "random" logic to determine whether or not the beastie will show up. So that it can only ever be at most 3 but will oftentimes be less than 3 (maybe even none?). But that doesn't need to happen in this pr.